src/index.js

```diff
@@ -12,25 +12,15 @@ import { render } from 'react-dom';
 
 import Button from './Button';
 
-render(
-	<h1>Hello World</h1>,
-	document.getElementById('app')
-)
-
-const name = 'Darren';
-const element = <h1>Hello {name}</h1>;
-
-render(
-	element,
-	document.getElementById('helloName')
-	);
-
-render(
-	<Button type="submit" />,
-	document.getElementById('appButton')
-	);
-
-render(
-	<Button onClick="{resetProduct}" />, 
-	document.getElementById('resetProductBtn')
-	);
+const rootElement = document.getElementById('appButton')
+
+function resetProduct() {
+  	document.getElementById('productLogo').src='img/DPW_Skull_Logo_Mask.jpg';
+
+	document.getElementById("eyesBlue").checked = false;
+	document.getElementById("eyesGreen").checked = false;
+	document.getElementById("eyesGrey").checked = false;
+		
+}
+
+render(<Button onClick={resetProduct}>Reset to original</Button>, rootElement)
```

Comment:
Does this function get called when you click the button @dpwit ?

Reply:
Yes mate, works a treat!